docs/development/building.md

```diff
@@ -37,123 +37,75 @@ git tag --sort=-creatordate
 Build a debug version of Santa:
 
 ```sh
-bazel build //Source/santa_driver
+bazel build //Source/santa:Santa
 ```
 
-Build a release (optimized) version of Santa:
+For developers who do not have access to Google's code signing certificate and
+provisioning profiles, use the `--define=SANTA_BUILD_TYPE=adhoc` flag. This will
+adhoc sign Santa and does not require provisioning profiles.
+
+Note: In order to run an adhoc signed Santa SIP must be disabled. See the
+running section below.
 
 ```sh
-bazel build //Source/santa_driver -c opt
+bazel build //Source/santa:Santa --define=SANTA_BUILD_TYPE=adhoc
 ```
 
-The output for these commands will be a `santa-driver.zip` file under
-`bazel-bin` which, when extracted, will contain all of Santa and should be
-installed under `/Library/Extensions`. However, if you're working on Santa and
-want a quick way to reload everything, see the next section.
-
 #### Running
 
 When working on Santa, it's useful to have a way to quickly reload all of the
 Santa components. For this reason, there's a special rule in the Santa BUILD
-file that will compile a new santa-driver, unload Santa if it's running, install
-the new Santa in the right place and attempt to load it.
-
-On macOS 10.11+ System Integrity Protection (SIP) prevents loading of kernel
-extensions that are not signed by an Apple KEXT signing certificate. To be able
-to load and test a non-release version of Santa, SIP will have to be configured
-to allow non-Apple KEXT signing certificates.
-
-__This is only to be done a machine that is actively developing, unloading and
-loading kernel extensions.__
-
-1.  Boot into Recovery Mode: Reboot and hold down `command+r`
-2.  From the utilities menu select `Terminal`
-3.  Disable the KEXT feature of SIP: `csrutil enable --without kext`
-4.  Reboot
+file that will build Santa, unload Santa if it's running, and install the new
+Santa in the right place and attempt to load it.
 
-You should now be able to load and run a non-release version of Santa.
-
-Build and run a debug version of Santa.
+Non-adhoc builds of Santa can only be run by Google developers.
 
 ```sh
 bazel run :reload
 ```
 
-Build and run a release version of Santa.
+Non-Google developers can use an adhoc build to run development builds of Santa.
+System Integrity Protection (SIP) will need to be disabled in order to run an
+adhoc build.
 
-```sh
-bazel run :reload -c opt
-```
+**This is only to be done a machine that is actively developing Santa.**
+
+1.  Boot into Recovery Mode: Reboot and hold down `command+r`
```

Comment:
Should we document the Apple Silicon version?

Reply:
Oh, it's just power off and then keep holding power for 10s until it goes into recovery

Reply:
https://support.apple.com/guide/mac-help/macos-recovery-a-mac-apple-silicon-mchl82829c17/mac